.github/workflows/eden.yml

```diff
@@ -1,105 +1,26 @@
-          ./eden setup -v debug --grub-options='set_global dom0_extra_args "$dom0_extra_args eve_install_zfs_with_raid_level "'
-      - name: clean-docker
-        run: docker system prune -f -a
-      - name: run
-        run: EDEN_TEST_STOP=n ./eden test ./tests/workflow -v debug
-      - name: Collect info
-        if: ${{ failure() }}
-        uses: ./.github/actions/collect-info
-        with:
-          working-directory: ${{ github.workspace }}
-      - name: Collect logs
-        if: ${{ always() }}
-        run: |
-          ./eden log --format json > trace.log || echo "no log"
-          ./eden info --format json > info.log || echo "no info"
-          ./eden metric --format json > metric.log || echo "no metric"
-          ./eden netstat --format json > netstat.log || echo "no netstat"
-          cp dist/default-eve.log console.log || echo "no device log"
-          docker logs eden_adam > adam.log 2>&1 || echo "no adam log"
-      - name: Log counting
-        if: ${{ always() }}
-        run: |
-          echo "::group::Total errors"
-          echo "$(jq '.severity' trace.log|grep err|wc -l)"
-          echo "::endgroup::"
-          echo "::group::Errors by source"
-          echo "errors by source: $(jq -s 'map(select(.severity|contains("err")))|group_by(.source)|map({"source": .[0].source, "total":length})|sort_by(.total)|reverse[]' trace.log)"
-          echo "::endgroup::"
-          echo "::group::Error log content duplicates"
-          echo "$(jq -s 'map(select(.severity | contains("err")))|group_by(.content)|map(select(length>1))' trace.log)"
-          echo "::endgroup::"
-          echo "::group::Error log function filename duplicates"
-          echo "$(jq -s 'map(select(.severity | contains("err")))|group_by(.filename)|map(select(length>10))|map({"source": .[0].source, "filename": .[0].filename, "function": .[0].function, "content": [.[].content], "total":length})|sort_by(.total)| reverse[]' trace.log)"
-          echo "::endgroup::"
-          echo "::group::Segfaults"
-          echo "$(jq -s 'map(select(.content | contains("segfault at")))' trace.log)"|tee segfaults.log
-          [ "$(jq length segfaults.log)" -gt 0 ] && echo "::warning::segfaults found, you can see them in Log counting->Segfaults section"
-          echo "::endgroup::"
-      - name: Store raw test results
-        if: ${{ always() }}
-        uses: actions/upload-artifact@v3
-        with:
-          name: eden-report-tpm-${{ matrix.tpm }}-${{ matrix.fs }}
-          path: |
-              ${{ github.workspace }}/eve-info.tar.gz
-              ${{ github.workspace }}/trace.log
-              ${{ github.workspace }}/info.log
-              ${{ github.workspace }}/metric.log
-              ${{ github.workspace }}/netstat.log
-              ${{ github.workspace }}/console.log
-              ${{ github.workspace }}/adam.log
+  test_suite_pr:
+    if: github.event.review.state == 'approved'
+    uses: ./eden/.github/workflows/test.yml@master
```

Comment:
Should we split the PR into two if you would like to modify the file you point onto here, but it doesn't exists in master branch?

Reply:
this file exists in master, only change here is runners for forks will be ubuntu :D

Reply:
syntax is GHA is tricky, my intention was to use local action, fixed to use just that. GHA seems to work, I'll wait for it to finish.

Reply:
[It](https://github.com/lf-edge/eden/actions/runs/6587792704/job/17898759398?pr=909#step:2:9) looks like another problem. It is really tricky

Reply:
That's just my poor bash-ninja skills :D

Reply:
Probably 
```
[[ "${{ github.event.repository.full_name}}" = "lf-edge/eve" ]] || [[ "${{ github.event.repository.full_name}}" = "lf-edge/eden" ]]
```

Reply:
Seems to be [working](https://github.com/lf-edge/eden/actions/runs/6588817373/job/17902163780)

Reply:
I think it'll make sense to include similar thing to eden_setup to speed up process in eden repository